easybuild/easyconfigs/p/PyTorch-extensions/PyTorch-extensions-2019.01-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'PythonBundle'
+
+name = 'PyTorch-extensions'
+version = '2019.01'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/rusty1s'
+description = """This is a bundle of extension libraries for PyTorch written by Matthias Fey."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+builddependencies = [('binutils', '2.31.1')]
+dependencies = [
+    ('Python', '3.7.2'),
+    ('PyTorch', '1.1.0', versionsuffix),
+    ('scikit-learn', '0.20.3'),
+    ('scikit-image', '0.15.0', versionsuffix),
+    ('numba', '0.43.1'),
+    ('h5py', '2.9.0'),
+    ('RDFlib', '4.2.2'),
+    ('google-drive-downloader', '0.4'),
+    ('plyfile', '0.7'),
```

Comment:
I know it was suggested otherwise on the EasyBuild mailing list, but I would include both `google-drive-downloader` and `plyfile` via `exts_list` rather than adding dedicated easyconfig files for them...

It seems unlikely to me that those stand-alone installations would be useful to researchers, and they only add up to the pile of modules we already have.

Reply:
Agreed.